src/table/virtualized-table/hooks/use-dynamic-row-height.ts

```diff
@@ -42,6 +44,8 @@ const useDynamicRowHeight = ({
   columns,
   boldText,
   gridState,
+  isSnapshot,
+  viewService, 
```

Comment:
We don't need to pass it here. We can get viewService through 
useContextSelector(TableContext, (value) => value.baseProps)